docs/source/release_notes/1.13.rst

```diff
@@ -0,0 +1,67 @@
+============
+OpenKAT 1.13
+============
+
+We are replacing the uWSGI server in rocky with Granian. uWSGI doesn't seem to
+be well maintained anymore and we run into a bug that exists for a long time and
+doesn't get fixed. Granian is new application server that is better maintained
+and also has good performance. Granian is also written in Rust instead of C so
+should also have better security.
+
+If you have made any configuration changes or are using rocky on port 8443 you
+need to make sure to configure Granian correctly as it will listen on 127.0.0.1
+port 8000 by default. To not break anything in 1.13 we will support using both
```

Comment:
Maybe add explicitly that we listen on `0.0.0.0` in the old situation, which is no longer the case with Granian (if I remember correctly).